gnovm/pkg/gnolang/ownership.go

```diff
@@ -96,6 +96,9 @@ type Object interface {
 	GetOwnerID() ObjectID
 	SetOwner(Object)
 	GetIsOwned() bool
+	// GetIsReal determines the reality of an Object.
+	// During a transaction, the object is fake, but becomes real upon successful completion, making it persisted and verifiable.
+	// This concept reflects a metaphysical understanding, where proof and persistence define an object's reality.
 	GetIsReal() bool
```

Comment:
I think we need to decide on whether to have long docs on the interface or on the method (of ObjectInfo). The other one should refer to the one with the long doc. I'm fine either way, but I slightly lean towards having the long doc here instead of ObjectInfo, as we mostly use the Object interface instead of ObjectInfo.

In either case, I think the longer doc, whichever it is, that should additionally have the note currently in the func doc of `ObjectInfo.GetIsReal`:

```
// NOTE: does not return true for new reals.
```